playwright/tests/rex-test/rex.behaviorspec.ts

```diff
@@ -411,3 +411,126 @@ test('C242991 special characters are escaped in slug', async ({ page, isMobile }
   expect(await Toc.SectionName()).toBe('2.1 Dlaczego badania są ważne?')
   expect(await Toc.ChapterName()).toBe('2         Prowadzenie badań')
 })
+
+test('MH page dropdown filters', async ({ page, isMobile }) => {
+  test.skip(isMobile as boolean, 'test only desktop resolution')
+
+  // GIVEN: Open Rex page
+  const BookPage = new ContentPage(page)
+  const path = '/books/introduction-anthropology/pages/7-introduction'
+  await BookPage.open(path)
```

Comment:
by convention (and standard TS linting) all variables should begin with a lower case letter